scripts/ntx_data/ntx_get_krea_count.py

```
@@ -0,0 +1,161 @@
+from scripts.ntx_data.utils import processed_data_dir
```

Pylint (4.0.2)

[error] No name 'processed_data_dir' in module 'scripts.ntx_data.utils'

(E0611)